classes/DailyTask.php

```diff
@@ -54,10 +60,11 @@ class DailyTask {
         DailyTask::ACTIVITY_MISSIONS => 'Missions',
         DailyTask::ACTIVITY_TRAINING => [
             DailyTask::SUB_TASK_SKILL => 'points',
-            DailyTask::SUB_TASK_GEN => 'points',
+            DailyTask::SUB_TASK_ATTRIBUTES => 'points',
             DailyTask::SUB_TASK_JUTSU => 'levels',
         ],
```

Comment:
To make this usable, we could take off the sub tasks and make it a flat stat-based task - train any skill or attribute by XYZ points. Then we don't assign the task if player is already at total stat cap

Reply:
I like this as a change. I'm going to keep the jutsu training as a task set to break up monotony of the task-type and provide an alternative if skills are capped on when the task reroll happens.

Reply:
Will complete this in next PR